firefox-ios/firefox-ios-tests/Tests/SyncIntegrationTests/xcodebuild.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, log, **kwargs):
         self.log = log
 
     def install(self, boot=True):
-        command = "find ~/Library/Developer/Xcode/DerivedData/Client-*/Build/Products/Fennec-* -type d -iname 'Client.app'"
+        command = "find ~/Library/Developer/Xcode/DerivedData/Client-*/Build/Products/Fennec_Testing-* -type d -iname 'Client.app'"
```

Comment:
This is the code for Sync Integration Tests. Does the ExperimentInegration test depend on this python file?

Reply:
Yes it does. If this is a problem I can create my own file based off of this.

Reply:
yeah you may need a different file because the builds are called that way for our tests